src/components/config/channels/whatsapp/Config.vue

```diff
@@ -56,6 +56,18 @@
       </unnnic-tab>
       <skeleton-loading v-else />
     </div>
+    <div class="config-whatsapp__insights">
+      <div class="config-whatsapp__insights__title">Insights</div>
+      <div class="config-whatsapp__insights__about">
+        <p>
+          Veja dados mais detalhados relacionados a envio, entrega e leitura dos modelos de mensagem
+          disparados.
```

Comment:
Is it intentional to have the final Portuguese text without considering its translatability?